packages/shoreline/src/components/collection/collection-view.tsx

```diff
@@ -57,19 +57,14 @@ export const CollectionView = forwardRef<HTMLDivElement, CollectionViewProps>(
       )
     }
 
+    const handleActionByStatus: Record<string, (() => void) | undefined> = {
```

Comment:
What if I pass the function "onError" and the status is `empty`? In this case empty state will have an action, right?

Reply:
No. The action button will only appear if the corresponding action function has been passed.

If it passes onError and not onEmpty, it renders the action button only in status=error
If it passes onError and onEmpty, renders action button in both statuses
If you pass onEmpty and not onError, it renders the action button only in status=empty
If neither onError nor onEmpty passes, the button will not be rendered in any state

Reply:
ok